docs/src/pages/guides/styled-engine/styled-engine.md

```diff
@@ -33,6 +37,39 @@ By default, `@mui/material` has `@mui/styled-engine` as a dependency, but you ca
  }
 ```
 
+### npm
+
+As the package aliases do not work with npm at this moment, you need to update you bundler's config to add this alias. Here is an example of how you can do it, if you use `webpack`:
+
+**webpack.alias.js**
+
+```diff
+ module.exports = {
+   //...
++  resolve: {
++    alias: {
++      '@mui/styled-engine': '@mui/styled-engine-sc'
++    },
++  },
+ };
+```
+
+If you are using TypeScript, you will need to also update the `tsconfig`.
```

Comment:
Seems like the TypeScript docs refer to the config as TSConfig: https://www.typescriptlang.org/tsconfig

```suggestion
If you are using TypeScript, you will need to also update the TSConfig.
```